tests/snuba/api/endpoints/test_organization_events_span_metrics.py

```diff
@@ -1080,6 +1082,144 @@ def test_http_response_rate(self):
+            1,
+            timestamp=self.three_days_ago,
+            tags={"transaction": "/api/0/projects/", "span.description": "Non-regressed"},
+            project=self.project.id,
+        )
+        self.store_span_metric(
+            1,
+            timestamp=self.min_ago,
+            tags={"transaction": "/api/0/projects/", "span.description": "Non-regressed"},
+            project=self.project.id,
+        )
+
+        response = self.do_request(
+            {
+                "field": [
+                    "span.description",
+                    f"regression_score(span.self_time,{int(self.two_min_ago.timestamp())})",
+                ],
+                "query": "transaction:/api/0/projects/",
+                "dataset": "spansMetrics",
+                "orderby": [
+                    f"-regression_score(span.self_time,{int(self.two_min_ago.timestamp())})"
+                ],
+                "start": (self.six_min_ago - timedelta(minutes=1)).isoformat(),
+                "end": before_now(minutes=0),
+            }
+        )
+
+        assert response.status_code == 200, response.content
+        data = response.data["data"]
+        assert len(data) == 2
+        assert [row["span.description"] for row in data] == ["Added span", "Non-regressed"]
+
+    def test_regression_score_removed_span(self):
+        # This span only exists before the breakpoint
+        self.store_span_metric(
+            100,
+            timestamp=self.six_min_ago,
+            tags={"transaction": "/api/0/projects/", "span.description": "Removed span"},
+            project=self.project.id,
+        )
+
+        # This span stays the same
+        self.store_span_metric(
+            1,
+            timestamp=self.three_days_ago,
+            tags={"transaction": "/api/0/projects/", "span.description": "Non-regressed"},
+            project=self.project.id,
+        )
+        self.store_span_metric(
+            1,
+            timestamp=self.min_ago,
+            tags={"transaction": "/api/0/projects/", "span.description": "Non-regressed"},
+            project=self.project.id,
+        )
+
+        response = self.do_request(
+            {
+                "field": [
+                    "span.description",
+                    f"regression_score(span.self_time,{int(self.two_min_ago.timestamp())})",
+                ],
+                "query": "transaction:/api/0/projects/",
+                "dataset": "spansMetrics",
+                "orderby": [
+                    f"-regression_score(span.self_time,{int(self.two_min_ago.timestamp())})"
+                ],
+                "start": (self.six_min_ago - timedelta(minutes=1)).isoformat(),
+                "end": before_now(minutes=0),
+            }
+        )
+
+        assert response.status_code == 200, response.content
+        data = response.data["data"]
+        assert len(data) == 2
+        assert [row["span.description"] for row in data] == ["Non-regressed", "Removed span"]
```

Comment:
May be worth asserting that the regression score is negative here as this gives us a way to filter on the regression score for regressions/improvements.